app/src/main/res/layout/fragment_result_list.xml

```diff
@@ -160,6 +161,8 @@
 		<androidx.recyclerview.widget.RecyclerView
 			android:id="@+id/recycler"
 			android:layout_width="match_parent"
-			android:layout_height="match_parent" />
+			android:layout_height="match_parent"
+			android:clipToPadding="false"
+			android:paddingBottom="66dp" />
 	</LinearLayout>
-</androidx.coordinatorlayout.widget.CoordinatorLayout>	<!-- app:layout_scrollFlags="scroll|exitUntilCollapsed|snap" -->
\ No newline at end of file
+</androidx.coordinatorlayout.widget.CoordinatorLayout>    <!-- app:layout_scrollFlags="scroll|exitUntilCollapsed|snap" -->
```

Comment:
why there is a commented line here?

Reply:
is only a placeholder for removed transition